### ISSUE TITLE
[Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See <URL>

### ISSUE THREAD

**Issue by VjeraTurk** [igorxciv/angular-flip-flip#66]
I'm running a simple Angular app with ngx-flip-flip used as in official example. 

I'm getting this error in Chromium Version 92.0.4515.131 (Official Build) Arch Linux (64-bit):
**[Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See <URL>**

And this warning in Firefox Browser 91.0 (64-bit ) Mozilla Firefox for Manjaro Linux:
**Ignoring ‘preventDefault()’ call on event of type ‘wheel’ from a listener registered as ‘passive’.**

I notice not every scroll action is picked up by the browser.
If there's any other info I can provide please let me know.

I'm using typescript, and had declared 
```
import { Provider } from "@angular/core";

declare module "@angular/core" {
  interface ModuleWithProviders<T = any> {
      ngModule: Type<T>;
      providers?: Provider[];
  }
}
```
to make the package work for me.

Thank you and sorry for posting an issue (in case it's not really one)

**Comment by VjeraTurk** [comment]
I noticed it is also happening at your official page https://ngx-flip-flip.netlify.app/, so it is browser connected.

**Comment by igorxciv** [comment]
@VjeraTurk sorry for such a long silence. Thanks for reporting this issue!
Fixed in the scope of #68 
I've released a new version `0.0.5`. Could you try to update, please?
